rgw/v2/tests/s3cmd/test_bucket_stats.py

```diff
@@ -60,7 +60,7 @@ def test_exec(config, ssh_con):
             log.info(f"Bucket {bucket_name} created")
             utils.exec_shell_cmd(f"fallocate -l 25m obj25m")
             object_name = (
-                f"s3://{bucket_name}/encyclopedia/space & universe/.bkp/journal$i"
+                f"s3://{bucket_name}/encyclopedia/space.and.universe/.bkp/journal$i"
             )
```

Comment:
@TejasC88 object name : f"s3://{bucket_name}/encyclopedia/space & universe/.bkp/journal$i" is expected as per BZ , this is a custer BZ
and error observed in log is expected

Reply:
thanks @anrao19 for catching this , I had not gone through the BZ , the fix is missing in 6.0 , have cloned the 5.x BZ for 6.x : https://bugzilla.redhat.com/show_bug.cgi?id=2181424

Reply:
Have reverted  my changes , and here is the failure log for the stats script:
http://magna002.ceph.redhat.com/ceph-qe-logs/tejas/bucket_stats_s3cmd.txt

Reply:
@anrao19 , hey, was this bug not fixed, is it a regression?

Reply:
@TejasC88 , as per the log: http://magna002.ceph.redhat.com/ceph-qe-logs/tejas/bucket_stats_s3cmd.txt
we are not observing the issue mentioned in : https://bugzilla.redhat.com/show_bug.cgi?id=2181424

looks like its differnt issue .

@viduship , yes the bug is fixed. we are no longer observing this issue